Source/WebCore/platform/mac/PlatformPasteboardMac.mm

```diff
@@ -375,7 +375,7 @@ static String typeIdentifierForPasteboardType(const String& pasteboardType)
         return 0;
 
     NSArray *urlWithTitle = @[ @[ urlString ], @[ pasteboardURL.title ] ];
-    NSString *pasteboardType = [NSString stringWithUTF8String:WebURLsWithTitlesPboardType];
+    NSString *pasteboardType = [NSString initWithCString:WebURLsWithTitlesPboardType encoding:NSASCIIStringEncoding];
```

Comment:
Isn't this missing a `[NSString alloc]`? Also, seems we should adoptNS() to avoid leaking now.

Reply:
Maybe you meant to use https://developer.apple.com/documentation/foundation/nsstring/1497310-stringwithcstring?language=objc ?